src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingGuardTower.java

```diff
@@ -67,6 +67,12 @@ public int getClaimRadius(final int newLevel)
         return Math.max(0, newLevel - 1);
     }
 
+    @Override
+    protected int getBonusHealth()
+    {
+        return BONUS_HP_SINGLE_GUARD + getBuildingLevel() * BONUS_HEALTH_PER_LEVEL;
```

Comment:
Leave Bonus health private and call `super.getBonusHealth()` here instead

Reply:
ye